bin/install.js

```diff
@@ -49,13 +49,7 @@ if (
 }
 
 if ('react-scripts' in projectDependencies || 'next' in projectDependencies) {
-  newDevDependenciesToInstall.push(
-    'postcss-styled-syntax',
-    'stylelint',
-    'stylelint-config-css-modules',
-    'stylelint-config-recommended',
-    'stylelint-config-recommended-scss',
-  );
```

Comment:
this dependencies are now added in the installation of stylelint-config-upleveled